packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -496,7 +496,7 @@ const DateRangeCalendar = React.forwardRef(function DateRangeCalendar<TDate>(
           {calendars === 1 ? (
             <PickersCalendarHeader
               views={['day']}
-              openView={'day'}
+              view={'day'}
```

Comment:
`PickersCalendarHeader` is exposed via the theme
But the typing only contains `classes` so I would not count this one as a breaking change